packages/client/src/comp/states/Minnesota.tsx

```diff
@@ -29,20 +22,25 @@ export const Minnesota = () => {
 
   return <SignatureBase<MinnesotaInfo>enrichValues={enrichValues}>
     <p>Minnesota requires voters to confirm their identify using one of the following types of identification</p>
-    <Select ref={idTypeRef} label='Identification Type' defaultValue='Select' {...{required: true}}>
-      <Option key={0} hidden={true}/>
-      {[...minnesotaIdentityType].sort().map((value, key) => {
-        return <Option value={value} key={key+1} label={value}/>
-      })}
-    </Select>
-    <p>Enter the relevant information based on your choice above.  If &apos;None&apos; 
+    <Select
+      label='Identification Type'
+      value={idType}
+      options={[...minnesotaIdentityType]}
+      onChange={v => setIdType(v)}
+      {...{required: true}}
+    />
+    <p>Enter the relevant information based on your choice above.  If &apos;None&apos;
       please confirm by typing &apos;None&apos;:
     </p>
     <BaseInput
       id='identityData'
-      ref={idDataRef}
       label='Identity Information'
+      value={idData}
+      onChange={e => setIdData(e.currentTarget.value)}
       required={true}
+      // HTML patterns are never case insensitive, manually creating a regexp
+      // that allows users to ignore case here
+      pattern={idType === 'None' ? '^(\\s*)(N|n)(O|o)(N|n)(E|e)(\\s*)$' : undefined}
```

Comment:
In the past (before this PR) we didn't check if users have typed 'None' (although the UI tells them to), I think it would be better to replace this input with a `<TogglableDropdown/>` (like we did with Kansas), and ask users to fill a checkbox to confirm when sending with no identifications.

Reply:
Good idea -- let's make that a new PR.